Server/src/overseer/teams/__init__.py

```
@@ -0,0 +1,24 @@
+    MARKET_INTELLIGENCE,
+    INVENTORY_OPERATIONS,
+)
+
+__all__ = [
+    "TEAM_SPECS",
+    "CONTENT_OPERATIONS",
+    "CUSTOMER_SUPPORT",
+    "INVENTORY_OPERATIONS",
+    "LEAD_GENERATION",
+    "MARKET_INTELLIGENCE",
+]
```

Ruff (0.16.3)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)